osu.Game/Overlays/Settings/SettingsSection.cs

```diff
@@ -19,10 +20,10 @@ public abstract class SettingsSection : Container, IHasFilterableChildren
         protected override Container<Drawable> Content => FlowContent;
 
         public abstract Drawable CreateIcon();
-        public abstract string Header { get; }
+        public abstract LocalisableString Header { get; }
 
         public IEnumerable<IFilterable> FilterableChildren => Children.OfType<IFilterable>();
-        public virtual IEnumerable<string> FilterTerms => new[] { Header };
+        public virtual IEnumerable<string> FilterTerms => new[] { Header.ToString() };
```

Comment:
Just to make note of this because I didn't see this mentioned anywhere in this thread: this in particular will mean that searching will still only work in English despite the visible strings not necessarily being in English any more.